lib/reporters/tap.js

```diff
@@ -28,7 +27,6 @@ exports = module.exports = TAP;
 function TAP(runner) {
   Base.call(this, runner);
 
-  var self = this;
   var n = 1;
   var passes = 0;
   var failures = 0;
```

Comment:
L30-L32 seem unnecessary. How is this different from `runner.stats` (from `Base`)?